src/components/FooterLayout/footerlayout.css

```diff
@@ -0,0 +1,10 @@
+.FooterLayout {
```

Comment:
it is good practice to name classes like `footer-layout`

Reply:
same for other classes

Reply:
fixed